src/Sulu/Bundle/ContentBundle/Preview/PreviewRenderer.php

```diff
@@ -34,20 +36,30 @@ class PreviewRenderer
      */
     private $controllerResolver;
 
-    public function __construct(ActiveTheme $activeTheme, ControllerResolverInterface $controllerResolver, WebspaceManagerInterface $webspaceManager)
-    {
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    public function __construct(
+        ActiveTheme $activeTheme,
+        ControllerResolverInterface $controllerResolver,
+        WebspaceManagerInterface $webspaceManager,
+        RequestStack $requestStack
+    ) {
         $this->activeTheme = $activeTheme;
         $this->controllerResolver = $controllerResolver;
         $this->webspaceManager = $webspaceManager;
+        $this->requestStack = $requestStack;
     }
 
     /**
      * renders content with the real website controller
-     * @param StructureInterface $content
+     * @param Page $content
      * @param bool $partial
      * @return string
      */
-    public function render(StructureInterface $content, $partial = false)
+    public function render(Page $content, $partial = false)
```

Comment:
Why did you change this? The `DefaultController` does also take the `StructureInterface`.